### DISCUSSION TITLE
config example

### DISCUSSION THREAD

**Discussion by btopro** [cifkao/html-midi-player#4] [Q&A] [answered]
Do you have any examples of modifying the config options in the visualizer? I see it in the docs and I'm supplying an object to the visualizer tag but it doens't seem to care about my options:
visualizerElement is a stored reference to the `midi-visualizer` tag
```js
    this.noteHeight = 40;
    this.pixelsPerTimeStep = 600;
    this.minPitch = 300;
    this.visualizerElement.config = {
          noteHeight: this.noteHeight,
          pixelsPerTimeStep: this.pixelsPerTimeStep,
          minPitch: this.minPitch
    };
```

**Comment by cifkao** [comment]
Interesting, I don't see how it could fail. It works from me when I try it from the console... Can you check if the property is being set correctly?

**Comment by btopro** [comment, marked as answer]
ah I see the config is different per type of visual based on the class it has it has other props. I see the values are going through they just dont always influence the visual bc they don't all use those three in my example
